packages/create-modular-react-app/package.json

```diff
@@ -20,6 +20,7 @@
     "build": "babel --root-mode upward src --out-dir build --extensions .ts --ignore 'src/**/*.test.ts'"
   },
   "dependencies": {
+    "@schemastore/package": "0.0.6",
```

Comment:
Hmm - this is technically a false positive. Since this is a "types-only" package then we don't actually want to add it as a dependency. Enabling `devDependencies` and then moving this there should fix that problem.

Reply:
Makes sense - I'll move them to `devDependencies` and add inline exceptions

Reply:
As you noted before - we can move this to `devDependencies` and add `import type ...` instead.

Reply:
I've moved all `@schemastore/` dependencies to `devDependencies` and used `import type...` on the imports themselves.